content/source/docs/enterprise/upgrade/differences.html.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: enterprise2
+page_title: "Current vs. Legacy - Upgrading - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-upgrading-differences"
+---
+
+# Differences Between Current and Legacy Terraform Enterprise
+
+Terraform Enterprise began as part of the discontinued Atlas suite, and today's version is a ground-up redesign. If you've previously used the legacy version of Terraform Enterprise, we hope you enjoy the new version's improvements.
```

Comment:
and today's version is a redesign purpose-built for using Terraform in a team setting.

Reply:
👍

Reply:
I'd add some honesty in here about the unfortunate nature that we have to remove some features. "we hope you enjoy the new version's improvements. Unfortunately, as part of this redesign, some features have been removed and others are not yet implemented."

Reply:
👍